drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -35,6 +35,8 @@ class Hmis::Hud::Client < Hmis::Hud::Base
   has_many :income_benefits, through: :enrollments
   has_many :disabilities, through: :enrollments
   has_many :health_and_dvs, through: :enrollments
+  has_many :youth_education_statuses, **hmis_relation(:PersonalID, 'YouthEducationStatus'), dependent: :destroy
+  has_many :employment_educations, **hmis_relation(:PersonalID, 'EmploymentEducation'), dependent: :destroy
```

Comment:
Oh whoops, one more thing I missed @ZachCMP.
```suggestion
  has_many :youth_education_statuses, through: :enrollments
  has_many :employment_educations, through: :enrollments
```


Based on the below comment @eanders  I think we should stick to going through enrollments, although I can't explain exactly why that is..

https://github.com/greenriver/hmis-warehouse/blob/bfc4e46b752da232c08a074dd8242865c92ebff0/app/models/grda_warehouse/hud/client.rb#L131-L135

Reply:
Gotcha. I went ahead and committed it

Reply:
Yeah, I'm not 100% sure you need to for the HMIS, but if the enrollment isn't there, the other items don't make sense (they only work in the context of the enrollment, and have a required EnrollmentID.